docs/conf.py

```diff
@@ -120,6 +120,13 @@
 
 # -- Options for HTML output -------------------------------------------
 
+# Set canonical URL from the Read the Docs Domain
+html_baseurl = os.environ.get("READTHEDOCS_CANONICAL_URL", "")
+
+# Tell Jinja2 templates the build is running on Read the Docs
+if os.environ.get("READTHEDOCS", "") == "True":
+    html_context["READTHEDOCS"] = True
```

Comment:
Do we need to define `html_context`? Looks like the RTD build doesn't find it currently

```suggestion
html_context = {}
# Tell Jinja2 templates the build is running on Read the Docs
if os.environ.get("READTHEDOCS", "") == "True":
    html_context["READTHEDOCS"] = True
```